src/data/repositories/Dhis2ConfigRepository.ts

```diff
@@ -1,17 +1,19 @@
-import { Permission } from "../../domain/entities/Permission";
 import { ConfigRepository } from "../../domain/repositories/ConfigRepository";
 import { D2Api } from "../../types/d2-api";
 import { cache } from "../../utils/cache";
 import { DataStoreStorageClient } from "../clients/storage/DataStoreStorageClient";
 import { Namespaces } from "../clients/storage/Namespaces";
 import { StorageClient } from "../clients/storage/StorageClient";
 import { Instance } from "../entities/Instance";
-import { PersistedConfig } from "../entities/PersistedConfig";
+import { defaultConfig, PersistedConfig } from "../entities/PersistedConfig";
 import { User } from "../entities/User";
-import { getD2APiFromInstance, getMajorVersion } from "../utils/d2-api";
+import { getD2APiFromInstance } from "../utils/d2-api";
+import { importTranslate, TranslatableText } from "../../domain/entities/TranslatableText";
+import _ from "lodash";
```

Comment:
Super minor comment, we tend to declare external dependencies at the beginning

Reply:
Typically, external dependencies at the beginning + blank line + local dependencies. Not enforced, but as a general rule.